### ISSUE TITLE
Markdown numbered lists clip tens digit when rendered

### ISSUE THREAD

**Issue by pglezen** [a85/POSTMan-Chrome-Extension#10987]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When I add ten or more items to a numbered markdown list for documentation, the tens digit is clipped when rendered.  So the rended tenth element looks like

```
...
9. ninth element
0. tenth element
1. eleventh element
...
```
All digits are visible in edit mode, in both *Classic Markdown* and *Postman beta* editors.  Only when saved or previewed is the tens digit clipped.

### Steps To Reproduce

1. Go to any documentation panel.
2. Create a numbered list with more than nine items
3. Save it

You should see the tens digit clipped when the numbered list is rendered.  For convenience, try adding the list below to a Postman markdown editor.

```
1. primum
2. secundum
3. tertium
4. quartum
5. quintum
6. sextum
7. septimum
8. octavum
9. nonum
10. decimum
11. undecimum
```

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/1967533/172883698-41aa4e71-882e-4369-a3ea-0f75a1a8d026.png)

### Operating System

macOS

### Postman Version

UI version: 9.20.3, Desktop Platform Version: 9.19.0

### Postman Platform

Postman App

**Comment by kamaraj-p** [comment]
Hi @pglezen, thanks for reporting this. We are working on resolving the issue and will update once the fix is deployed.

**Comment by kamaraj-p** [comment]
Hi @pglezen, the issue has been fixed and should be available on versions 9.30.0 onwards. Thanks again for reporting this.

**Comment by frcake** [comment]
Hello, this issue does not seem to be fixed in version:
```
You are up to date!
Postman v10.15.4 is the latest version.
Last checked: June 27, 2023 at 1:08 PM
```

A numbered list will be depicted like this:
![image](https://github.com/postmanlabs/postman-app-support/assets/17022122/cfc9d4d8-0386-4cf4-b84a-ea1b9486ce4b)

Although if I add some element above the list (which shouldn't change anything but I did it anyway..)
![image](https://github.com/postmanlabs/postman-app-support/assets/17022122/847854e5-450c-4d9b-8608-b275d40b23c0)

It *will* show the full number (compared to clipping the actual depiction) but the numbered list will start from 0 after reaching 9..
